### ISSUE TITLE
Unchangeable font size

### ISSUE THREAD

**Issue by BillyBouw** [spyder-ide/spyder#14678]
Subject of the issue
I'm new to Spyder, I have used the program several times but when I started it recently something seemed off. The font size of my file names and all the tabs are very big all of a sudden. Things I tried so far;

In tools -> preferences -> appearance the font size of the 'plain text' and 'rich text' can be adjusted, but this doesn't affect the font size I try to alter.
Changing DPI settings also doesn't seem to help.
I also tried adding [Platforms] WindowsArguments = dpiawareness=0 to the qt.conf file, still nothing changed.
Windows font settings, no impact.
See attached screenshot, where I highlighted the text I want to make smaller.
Your environment
Spyder 4 (anaconda 3)
Python 3.7.9

Steps to reproduce
When I boot

Expected behaviour
Font size normal/not so big

Actual behaviour
Enormous font size in menus and tabs

![image](https://user-images.githubusercontent.com/75724300/106579587-2a2ffe00-6541-11eb-9c77-c832d33b0e00.png)

**Comment by steff456** [comment]
Hi @BillyBouw,

Did you change the font size in your operating system? Or have you seen this issue with other applications different from Spyder? 
Also, what is the setup you have? i.e laptop + screen, or just laptop

**Comment by BillyBouw** [comment]
@steff456 Hi thank for responding!
Like I stated I tried to change the font settings in windows (settings > display) but this didn't change a thing. I use a laptop and a monitor, but tried on both screens with different settings in windows and in Spyder itself (DPI), marked text stays the same size. I also think its maybe somewhere in windows but I can only find a scaling up down function in setting>display, which like I said doesn't affect Spyder.
